oximeter/collector/src/bin/oximeter.rs

```diff
@@ -99,6 +105,12 @@ enum Args {
         /// The log-level.
         #[arg(long, default_value_t = Level::Info, value_parser = parse_log_level)]
         log_level: Level,
+
+        // TODO (https://github.com/oxidecomputer/omicron/issues/4148): This flag
+        // should be removed once single node functionality is removed.
+        /// Is `oximeter` connecting to a replicated ClickHouse cluster
+        #[clap(short, long, num_args = 0, action)]
+        replicated: bool,
```

Comment:
It seems like we can always run `oximeter standalone` against a single-node database. It's a test-only mode, and the whole point is being able to easily test a producer. That doesn't seem like an undue constraint.